### DISCUSSION TITLE
z2m: MQQT error: connack timeout

### DISCUSSION THREAD

**Discussion by eagles2703** [Koenkk/zigbee2mqtt#24066] [General]
Running into the error below and cannot troubleshoot it. Probably simple, but I am pretty new.

Error:
[2024-09-22 16:04:16] info: 	z2m: Connecting to MQTT server at mqtt://xxx.xxx.xx.xx:1883
[2024-09-22 16:04:46] error: 	z2m: MQTT error: connack timeout
[2024-09-22 16:04:46] error: 	z2m: MQTT failed to connect, exiting... (connack timeout)
[2024-09-22 16:04:46] info: 	z2m: Stopping zigbee-herdsman...

Here is my docker compose yaml:
zigbee2mqtt:
    container_name: zigbee2mqtt
    image: koenkk/zigbee2mqtt
    restart: unless-stopped
    volumes:
        - /opt/zigbee2mqtt/data:/app/data
        - /run/udev:/run/udev:ro
    ports:
        # Frontend port
         - 8080:8080
    environment:
        - TZ=US/Central
    devices:
        # Make sure this matched your adapter location
        - /dev/serial/by-id/usb-Nabu_Casa_SkyConnect_v1.0_48b38dcd159aed11b9865fdba7669f5d-if00-port0:/dev/ttyACM0

Here is my zigbee2mqtt config yaml:

homeassistant: true
permit_join: true
mqtt:
  base_topic: zigbee2mqtt
  server: mqtt://xxx.xxx.xx.xx:1883
  user: my_username (placeholder here)
  password: my_password (placeholder here)
serial:
  adapter: ember
  rtscts: false
  port: /dev/ttyACM0

**Comment by francisp2** [comment]
install e.g. MQTT Explorer, and see if you can connect to the MQTT Broker.
